graylog2-server/src/main/java/org/graylog2/rest/resources/tools/JsonTesterResource.java

```diff
@@ -98,7 +99,12 @@ private JsonTesterResponse testJsonExtractor(String testString,
             throw new BadRequestException("Invalid extractor configuration", e);
         }
 
-        final Map<String, Object> result = extractor.extractJson(testString);
+        final Map<String, Object> result;
+        try {
+            result = extractor.extractJson(testString);
+        } catch (IOException e) {
+            throw new BadRequestException("Failure running JSON extractor", e);
```

Comment:
If we were unable to extract JSON, the UI was previously showing this:
![image](https://user-images.githubusercontent.com/886541/132462762-6e9b5b4a-524e-4126-a294-eb10e7f6074f.png)

Now we are showing an error overlay: 
![image](https://user-images.githubusercontent.com/886541/132462971-f56059c2-bb61-47ff-aa04-5bd7d6cb80c8.png)

What do you think about adding a bit more information what actually went wrong. Maybe just by including the actual error, or do you think this is too much information? Thinking about something along the lines of

```suggestion
            throw new BadRequestException("Failure running JSON extractor: " + e.getMessage(), e);
```

Reply:
I like it :+1: